dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -13,6 +13,8 @@
 //= require jquery-ui
 //= require video
 //= require bootstrap
+//= require react
```

Comment:
So I think adding this means we end up with two copies of React just in dashboard - one as react.js one bundled into application.js. I'm not too sure where the react.js version comes from. My first guess would be the react-rails gem, but their instructions explicitly state to require react in application.js, so I'm not sure. I'm going to remove the require react from application for now, and be sure to watch this PR once it's deployed.